README.md

```diff
@@ -144,6 +144,29 @@ especially in the case in which the exporter is external to the code it's monito
 
 For more detailed documentation on [Availability exporters](https://gitlab.cee.redhat.com/konflux/docs/documentation/-/blob/main/o11y/monitoring/availability_exporters.md?ref_type=heads)
 
+## Recording Rules
+
+Recording rules allow us to precompute frequently needed or computationally expensive expressions
+and save their result as a new set of time series. Recording rules are the go-to approach for
+speeding up the performance of queries that take too long to return. When other teams want to go
+with their own metrics format for exporters they need to adapt to desired metric form by
+translating it using a recording rule. 
+
+These recording rules should be put in the [rhobs/recording folder](https://github.com/redhat-appstudio/o11y/tree/main/rhobs). 
+
+The standard format is single availability metric `konflux_up` with labels `service` and `check`.
+Each time series will have the service and check labels for the name of the originating service
+and availability check it performed, respectively. The metric konflux_up will return either 0
+or 1 based on the availability of the component/service. In the case of the example exporter,
+it returns 1 if prometheus-appstudio-ds datasource exists in our grafana service else it returns 0.
```

Comment:
It's a bit ambiguous here. We should state it as a requirement that 1 should mean available and 0 should mean unavailable.

Reply:
@yftacherzog updated it, Please check.